src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -652,6 +652,32 @@ describe('<ResourceItem />', () => {
       });
     });
   });
+
+  describe('focused', () => {
+    it('removes the focus state when mousing out a focused item', () => {
+      const resourceItem = mountWithApp(
+        <ResourceListContext.Provider value={mockSelectModeContext}>
+          <ResourceItem id={itemId} url={url} />
+        </ResourceListContext.Provider>,
+      );
+      const wrapperDiv = resourceItem.find('div');
+      const className = 'ResourceItem selectable selectMode';
+      const focusClassName =
```

Comment:
Nit but you could add each class name into an array and .`join(' ')` for better readability.

Reply:
actually didn't need these in const.